app/src/standard/java/com/infomaniak/mail/firebase/ProcessMessageNotificationsWorker.kt

```diff
@@ -122,6 +123,16 @@ class ProcessMessageNotificationsWorker @AssistedInject constructor(
             if (!hasShownNotification) displayGenericNewMailsNotification()
             mailboxContentRealm.close()
         }
+
+        checkIfNotificationMessageUidIsInRealm(messageUid, mailboxContentRealm)
+
+        return@withContext workerResult
+    }
+
+    private fun checkIfNotificationMessageUidIsInRealm(messageUid: String, mailboxContentRealm: Realm) {
+        if (MessageController.getMessage(messageUid, mailboxContentRealm) == null) {
+            Sentry.captureMessage("Unable to find the message with the corresponding messageUid from the notification.")
+        }
```

Comment:
This is to be expected if we did not successfully manage to download the updates. In this case there's no point in checking and there is no point in sending a sentry event